### PR TITLE
Remove generated annotation simple

### PR DESCRIPTION
**Pull request by MysterAitch**
Simple, minimal, version of #2554 that doesn't contain any of the follow-up fixes/deprecations etc.

My main hesitation with this specific PR is that there are a substantial number of files which have whitespace-only changes. This is not improved by enabling lexical preservation.

### CODE REVIEW

**Comment by MysterAitch** [comment]
Here's a comparison without and without lexical preservation:

https://github.com/MysterAitch/javaparser/compare/MysterAitch:remove_generated_annotation_simple...MysterAitch:remove_generated_annotation_simple_with_lex_pres

File: javaparser-core/src/main/java/com/github/javaparser/JavaParser.java [hunk 1]
```diff
@@ -91,7 +89,7 @@ private GeneratedJavaParser getParserForProvider(Provider provider) {
         astParser.setTabSize(configuration.getTabSize());
         astParser.setStoreTokens(configuration.isStoreTokens());
         if (configuration.getLanguageLevel() != null) {
-            switch (configuration.getLanguageLevel()) {
+            switch(configuration.getLanguageLevel()) {
```
Comment (matozoid) [inline javaparser-core/src/main/java/com/github/javaparser/JavaParser.java:92]: Ah, I can see why you're building in that dirty detection. These files shouldn't be touched! The scope for the generators is: the AST nodes and the visitors. If you would like to extend that, please discuss a bit first so I'm not completely surprised :-D

File: javaparser-core/src/main/java/com/github/javaparser/JavadocParser.java [hunk 2]
```diff
@@ -48,33 +47,21 @@ public static Javadoc parse(JavadocComment comment) {
 
     public static Javadoc parse(String commentContent) {
         List<String> cleanLines = cleanLines(normalizeEolInTextBlock(commentContent, EOL));
-        int indexOfFirstBlockTag = cleanLines.stream()
-                .filter(JavadocParser::isABlockLine)
-                .map(cleanLines::indexOf)
-                .findFirst()
-                .orElse(-1);
+        int indexOfFirstBlockTag = cleanLines.stream().filter(JavadocParser::isABlockLine).map(cleanLines::indexOf).findFirst().orElse(-1);
```
Comment (matozoid) [inline javaparser-core/src/main/java/com/github/javaparser/JavadocParser.java:50]: Noooooo

**Comment by MysterAitch** [comment]
Closing in favour of #2704 (or whatever comes of that PR).

**Pull request closed without merging**